front/src/applications/stdcm/hooks/useStdcmResults.ts

```diff
@@ -67,23 +71,52 @@ const useStdcmResults = (
         await postPathProperties(pathPropertiesParams).unwrap();
 
       if (geometry && operational_points && zones) {
-        const operationalPointsWithUniqueIds = operational_points.map((op, index) => ({
-          ...op,
-          id: `${op.id}-${op.position}-${index}`,
-        }));
-
-        const suggestedOperationalPoints: SuggestedOP[] = formatSuggestedOperationalPoints(
-          operational_points,
-          geometry,
-          path.length
-        );
-
-        setPathProperties({
-          manchetteOperationalPoints: operationalPointsWithUniqueIds,
-          geometry,
-          suggestedOperationalPoints,
-          zones,
-        });
+        if (infraId) {
+          const trackIds = operational_points.map((op) => op.part.track);
+          const trackSections = await getEntities<TrackSectionEntity>(
+            infraId,
+            trackIds,
+            'TrackSection',
+            dispatch
+          );
+          const operationalPointsWithMetadata = operational_points.map((op) => {
+            const associatedTrackSection = trackSections[op.part.track];
+            const sncf = associatedTrackSection?.properties?.extensions?.sncf;
+
+            const metadata =
+              sncf && Object.values(sncf).every((value) => value !== undefined)
+                ? {
+                    lineCode: sncf.line_code!,
+                    lineName: sncf.line_name!,
+                    trackName: sncf.track_name!,
+                    trackNumber: sncf.track_number!,
+                  }
+                : undefined;
+
+            return {
+              ...op,
+              metadata,
+            };
+          });
+
+          const operationalPointsWithUniqueIds = operationalPointsWithMetadata.map((op, index) => ({
```

Comment:
You don't need to use operationalPointsWithMetadata to define the ops for the manchette, you can keep operational_points
```suggestion
          const operationalPointsWithUniqueIds = operational_points.map((op, index) => ({
```

Reply:
Done !